### PR TITLE
[1LP][RFR] Automating test_replication_remote_to_global_by_ip_pglogical

### PR DESCRIPTION
**Pull request by dgaikwad**
Signed-off-by: mnadeem92 <mnadeem@redhat.com>

Co-authored-by: Devidas Gaikwad <dgaikwad@redhat.com>
Co-authored-by: mnadeem92 <mnadeem@redhat.com>

{{ pytest: cfme/tests/test_replication.py::test_replication_remote_to_global_by_ip_pglogical --long-running  -svvv }}

### CODE REVIEW

File: cfme/tests/test_replication.py [hunk 1]
```diff
@@ -167,7 +170,11 @@ def test_replication_remote_to_global_by_ip_pglogical():
             2.
             3. Provider appeared in the Global.
     """
-    pass
+    remote_app, global_app = setup_replication
+    provider_app_crud(OpenStackProvider, remote_app).setup()
```
Comment (sbulage) [inline cfme/tests/test_replication.py]: **Suggestion:**
@dgaikwad you may use like this: 
```
provider = provider_app_crud(OpenStackProvider, remote_app)
provider.setup()
```

File: cfme/tests/test_replication.py [hunk 2]
```diff
@@ -145,9 +148,9 @@ def test_replication_delete_remote_from_global(setup_replication):
         region.replication.get_replication_status(host=remote_app.hostname)
 
 
-@pytest.mark.manual
+@pytest.mark.provider([OpenStackProvider])
```
Comment (sbulage) [inline cfme/tests/test_replication.py]: Remove `@pytest.mark.provider([OpenStackProvider])` Marker as we don't parameterize this test.

File: cfme/tests/test_replication.py [hunk 3]
```diff
@@ -145,9 +148,9 @@ def test_replication_delete_remote_from_global(setup_replication):
...
 @pytest.mark.tier(1)
-def test_replication_remote_to_global_by_ip_pglogical():
+def test_replication_remote_to_global_by_ip_pglogical(setup_replication, provider):
```
Comment (sbulage) [inline cfme/tests/test_replication.py]: Below comment will instantiate `provider` object hence no need to pass here, Pass `OpenStackProvider`.

**Comment by john-dupuy** [comment]
@dgaikwad I see no tests ran in PRT, can you check your PRT command and retest?

File: cfme/tests/test_replication.py [hunk 4]
```diff
@@ -167,7 +169,12 @@ def test_replication_remote_to_global_by_ip_pglogical():
             2.
             3. Provider appeared in the Global.
     """
-    pass
+    remote_app, global_app = setup_replication
+    provider = provider_app_crud(OpenStackProvider, remote_app)
+    provider.setup()
+
+    # Assert the provider is replicated to global appliance
+    assert provider.name in global_app.managed_provider_names, "Provider name not found"
```
Comment (sbulage) [inline cfme/tests/test_replication.py]: @dgaikwad Nice `AssertionError` message.

**Pull request merged**